services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_specs.py

```diff
@@ -1,18 +1,40 @@
 import json
+import logging
 from copy import deepcopy
-from typing import Dict, List, Optional, Union
+from typing import Any, Dict, Final, List, Optional, Union
 
 import yaml
 from fastapi.applications import FastAPI
-from models_library.service_settings_labels import ComposeSpecLabel, PathMappingsLabel
+from models_library.service_settings_labels import (
+    ComposeSpecLabel,
+    PathMappingsLabel,
+    SimcoreServiceLabels,
+    SimcoreServiceSettingLabelEntry,
+)
 from settings_library.docker_registry import RegistrySettings
 
+from ...modules.director_v0 import DirectorV0Client
 from ._constants import CONTAINER_NAME
 from .docker_service_specs import MATCH_SERVICE_VERSION, MATCH_SIMCORE_REGISTRY
+from .docker_service_specs.settings import get_labels_for_involved_services
+from .scheduler.events_utils import get_director_v0_client
 
 EnvKeyEqValueList = List[str]
 EnvVarsMap = Dict[str, Optional[str]]
 
+_NANO_UNIT: Final[int] = int(1e9)
+_MB: Final[int] = 1_048_576
+# defaults to use when these values are not defined or found
+# TODO: ANE -> SAN, PC: let's make these are sensible for when a service does not define them
+DEFAULT_LIMIT_MEMORY_BYTES: Final[int] = 100 * _MB
+DEFAULT_LIMIT_NANO_CPUS: Final[int] = int(0.1 * _NANO_UNIT)
```

Comment:
Can you check that these limits are the same the director-v0 returns?
and/or else this should be defined in a common location so that they do not diverge. Currently getting the service extra from director-v0 returns the limits. Shall we add the reservations?

Reply:
Outdate, no longer used. Defaults are provided by the catalog.